app/services/code_climate/repositories_summary_service.rb

```diff
@@ -15,6 +15,7 @@ def call
         invalid_issues_count_average: invalid_issues_count_average.round(2),
         wont_fix_issues_count_average: wont_fix_issues_count_average.round(2),
         open_issues_count_average: open_issues_count_average.round(2),
+        test_coverage_average: test_coverage_average&.round,
```

Comment:
why not `round(2)` as the ones above? Just to be consistent I guess..

Reply:
ahh that's a good point, I didn't realize that the other where counts and this one a percentage. I'm fine leaving it like this then 👌